automation/test.sh

```diff
@@ -429,18 +429,18 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} && -z ${label_filter}
   elif [[ $TARGET =~ vgpu.* ]]; then
     label_filter='(VGPU)'
   elif [[ $TARGET =~ sig-compute-realtime ]]; then
-    label_filter='(sig-compute-realtime)'
+    label_filter='(sig-compute-realtime) && !(software-emulation, SEV, SEVES)'
```

Comment:
@dhiller @brianmcarey Can you have a look at least on the changes in this file?

Reply:
Dumb q: I saw introduction of VSOCK label, however I don't see it used anywhere in this file - is that by intent? Do we want to use that label dynamically somewhere else?

Reply:
^^ @xpivarc

Reply:
@dhiller oh and yes I have a place where this will be used but as user of the test suite.

Reply:
Just one note on the SEV: as we currently don't have a functioning cluster there's no way to check whether this will work completely.